File: assets_rebalance_frontend/src/domain/fin-asset-bank.ts
import { Expose } from "class-transformer";
import { Entity } from "./entity";

export type FinAssetBank =  Entity & {
  name?: string; // required
  routing?: string; // required
  ispb?: string | null; // optional
  ispbName?: string | null; // optional
  iconBase64?: string | null; // optional
}

File: assets_rebalance_backend/src/Domain/FinAssetBank.cs
using System;

namespace assets_rebalance_backend.src.Domain;

public class FinAssetBank : Entity
{
    public required string Name { get; set; }
    public required string Routing { get; set; }
    public string? Ispb { get; set; }
    public string? IspbName { get; set; }
    public string? IconBase64 { get; set; }
}
